### PR TITLE
add utils getNetWorkID method and tests

### PR DESCRIPTION
**Pull request by pelsasser**
##### Description
adds simple util method that was in the service listener, and I was going to use in our demo projects. 

##### Checklist
- [x] Linter status: 100% pass
- [x] Changes don't break existing behavior
- [x] Tests coverage hasn't decreased

##### Affected core subsystem(s)
utils

### CODE REVIEW

File: test/Utils.test.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ import FakeProvider from 'web3-fake-provider';
 // Types
 import { ECSignature } from '@marketprotocol/types';
 import { Utils } from '../src';
+import { cat } from 'shelljs';
```
Comment (eswarasai) [inline test/Utils.test.ts]: @pelsasser -- Unused import

Comment (pelsasser) [reply test/Utils.test.ts]: good catch! thanks, fixing now.  Intellij sometimes sneaks some imports in there.

Comment (pelsasser) [reply test/Utils.test.ts]: sounds likely!

**Comment by pelsasser** [comment]
@eswarasai - good to merge?

**Comment by rojosnow** [comment]
@pelsasser @eswarasai Is there an issue for this PR? Want to make sure we capture the work in the iteration.

**Comment by pelsasser** [comment]
@rojosnow - there will be in a second ;)

**Comment by pelsasser** [comment]
@rojosnow - does putting the same fields on the PR not make it count if there is no issue?

**Pull request merged**